### ISSUE TITLE
Add Necessary Check for the Rating Button Group

### ISSUE THREAD

**Issue by jaykay12** [jaykay12/KitabGhar#19]
**Current Behavior:**
Rating Button Group returns an Integer value ranging from (1-5) and in Relational Table rating has NOT NULL constraint associated.
If in any case, No Rating is selected & Form is Submitted then, the portal generates **Error 500**

![screenshot from 2018-10-03 03-18-45](https://user-images.githubusercontent.com/13948542/46379039-40042e80-c6bb-11e8-921f-f3fd9e0d3cbe.png)

**Required Behavior:**
Graceful Submission of form & handling of such Errors

**Possible Methods:**
 - Using JS to keep Button `submitReview` inactive, until a rating is selected.
 - Using the REQUIRED attributed of CSS
 - Using JS to validate if the rating is selected or not, generating Browser Error Signal before Form Submission.

**Comment by AyahRamahi** [comment]
Can I work on this ?

**Comment by jaykay12** [comment]
Yes Sure @petervasil. Go ahead with this. I don't think @AyahRamahi is working on this issue.

**Comment by petervasil** [comment]
Thank you @jaykay12, I will fix it this week

**Comment by marekfe** [comment]
Hello. Could I also try to solve this problem to practice my skills ?
